.depcheckrc.yml

```diff
@@ -0,0 +1,44 @@
+# things that *are* used, that depcheck is wrong about
+ignores:
+  # 
+  # webapp deps
+  # 
+
+  - "@babel/runtime"
+  - "@fortawesome/fontawesome-free"
+  - "punycode"
```

Comment:
Curious why depcheck complains about this one :thinking: It seems to be a "regular" dependency as far as I can tell.

Reply:
yeah something about esm + importing an alternate entry point?